administration/src/index.tsx

```diff
@@ -11,7 +11,11 @@ import App from './App'
 import reportWebVitals from './reportWebVitals'
 
 const root = createRoot(document.getElementById('root')!)
-root.render(<App />)
+root.render(
+  <StrictMode>
```

Comment:
Warning: This might break other PRs which do not merge in whitelabel first.

Reply:
I think the probability is very low. If it's the case, then we'll have to fix it afterwards.